### DISCUSSION TITLE
第4节课笔记(22班)

### DISCUSSION THREAD

**Discussion by BarryYin** [InternLM/Tutorial#234] [笔记]
在这里提交链接

**Comment by miku8miku** [comment]
https://note.youdao.com/s/KAk2SQse

**Comment by lilxmx** [comment]
https://blog.csdn.net/L1142456160/article/details/135632018

**Comment by lankuohsing** [comment]
https://github.com/lankuohsing/InternLM_notes/blob/main/lecture_4/note_4.md

**Comment by liuwei16** [comment]
https://g5b4uupmby.feishu.cn/wiki/QFEKwUNrfi4bXukLAL9cDwEvnOc

**Comment by jiayaozhang** [comment]
https://github.com/jiayaozhang/InternLM_Notes/blob/main/Lecture4.md

**Comment by Jasimyn** [comment]
https://broadleaf-gemini-274.notion.site/Lecture4-XTuner-d6399774b18f4818ba8eeb1eceaf7a08?pvs=4

**Comment by lyyilin** [comment]
https://blog.csdn.net/qq_73668945/article/details/135563747

**Comment by mmb135** [comment]
https://www.yuque.com/r/note/61c8e43f-9851-4537-b667-088cd625bd7b

**Comment by cavedweller509** [comment]
https://github.com/BestAnHongjun/InternLM_Notes/blob/main/notes/lecture_4.md

**Comment by CDL0726** [comment]
https://github.com/CDL0726/InternLM_Notes/blob/main/Lecture4_XTuner%20Finetune

**Comment by funcfang** [comment]
https://www.yuque.com/funcfang/ke5qbq/hwr2idg3ootf74n0?singleDoc#

**Comment by linkch0** [comment]
https://linkch0.github.io/courses/InternLM/lecture/#l4-xtuner
